src/libumf_windows.c

```diff
@@ -33,14 +33,16 @@ void libumfInit(void) {
 
 INIT_ONCE init_once_flag = INIT_ONCE_STATIC_INIT;
 
+static void umfTearDownWrapper(void) { (void)umfTearDown(); }
```

Comment:
did you test it (e.g. on your fork)? pls paste a link to a passing build, if you have any

Reply:
I did not. 

Imho we can test it during nightly. Is simple enough patch to assume that it is for sure is working, and if not i can just then make another fix tomorrow.